tests/validate_revert_schema_test.php

```diff
@@ -61,8 +62,8 @@ public function test_missing_revert_schema()
 	{
 		$this->validateFile('tests/testFiles/migrations/missing_revert_schema.php', function($output)
 		{
-			/** @var PHPUnit_Framework_MockObject_MockObject $output */
-			$output->expects($this->once())
+			/** @var PHPUnit\Framework\MockObject\MockObject $output */
+			$output->expects($this->never())
```

Comment:
@Skouat Why did you change this to `never()`? it should be `once()` no? Because this is testing when `revert_schema` is missing and so the message should be shown once, not never. 

This is being brought up because it's being reported that EPV is not successfully checking for missing `revert_schema` now.

Reply:
... so that means there's still a problem with the missing revert schema test, not this unit test ;)

Reply:
#117

Reply:
> @Skouat Why did you change this to `never()`? it should be `once()` no? Because this is testing when `revert_schema` is missing and so the message should be shown once, not never.
> 
> This is being brought up because it's being reported that EPV is not successfully checking for missing `revert_schema` now.

It was in October, I don't really remember. 
As I'm not really familiar with phpunit, I think I had changed the parameter because of an error with phpunit that said once() didn't work with this test.

We will continue the chat on the opened ticket.